src/Core/Bases/A11yProperty.cs

```diff
@@ -147,7 +148,7 @@ private string GetBoundingRectangleText()
             if ((double)arr[2] < 0 || (double)arr[3] < 0)
             {
                 // the 3rd and 4th values in array are negative value, we need to show value in different format like l,t,w,h
-                text = string.Format("[l={0},t={1},w={2},h={3}]", arr[0], arr[1], arr[2], arr[3]);
+                text = string.Format(CultureInfo.InvariantCulture, "[l={0},t={1},w={2},h={3}]", arr[0], arr[1], arr[2], arr[3]);
```

Comment:
This is the ship-impacting change. By default, the numbers in the string should pick up culture-specific separators. With CultureInfo.InvariantCulture, all separators should be skipped.

Reply:
Does it then make sense to use the current culture?

Reply:
Upon further investigation, separators are not impacted by the current culture, but negative signs definitely are. This is [documented](https://docs.microsoft.com/en-us/dotnet/api/system.globalization.numberformatinfo.negativesign?view=net-5.0) and I confirmed it via a small test app. For a data file format, we should be using culture-agnostic formats both when reading and when writing, so a file created in culture X can be opened in culture Y.